backend/content/models.py

```diff
@@ -1,10 +1,35 @@
 from django.db import models
 from django.contrib.auth.models import User
 
+
 # Create your models here.
 
+class Genre(models.Model):
+    name = models.CharField(max_length=20, default="")
+
+    def __str__(self):
+        return self.name
+class Actor(models.Model):
+    name = models.CharField(max_length = 50, default="")
 
+    def __str__(self):
+        return self.name
 class Content(models.Model):
+    title = models.CharField(max_length=100, default="")
+    genres = models.ManyToManyField(
+        Genre,
+        blank=True
+    )
+    poster = models.CharField(default="", max_length=200)
+    overview = models.TextField(default="")
+    release_date = models.CharField(default="", max_length=15)
+    rate = models.DecimalField(default=0, max_digits=4, decimal_places=1)
+    cast = models.ManyToManyField(
+        Actor,
+        blank= True
+    )
+    director = models.CharField(max_length=100, default="")
+    ott = models.CharField(max_length=200, default="")
```

Comment:
how about making this a foreign key?

Reply:
For Director, I guess it can work too but I just made it to charfield since i thought there'll be 1 director per movie.
For ott, I also thought it'd be better to make it a foreign key but then thought it would make things complicated (because we have a ott structure already and we didn't separate membership in that one so we also need to do something about it) so i just kept it to charfield.